Makefile.am

```diff
@@ -42,8 +42,6 @@ TEST_EXTENSIONS = .int
 INT_LOG_COMPILER = $(srcdir)/script/int-log-compiler.sh
 INT_LOG_FLAGS = --simulator-bin=$(SIMULATOR_BIN)
 
-check-programs: $(check_PROGRAMS)
```

Comment:
Would you mind keeping the build target ? It doesn't hurt and you can have static code analysis of tests without requiring tests to be run.

Reply:
sure, will update PR